src/js/components/SiteSider.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import { Layout, Menu } from 'antd';
+import type { MenuProps } from 'antd';
+import { PieChartOutlined, SearchOutlined, SolutionOutlined } from '@ant-design/icons';
+import { Flex } from 'antd';
+import Icon from '@ant-design/icons';
+import BeaconSvg from './Beacon/BeaconSvg';
+import { buildQueryParamsUrl } from '@/utils/search';
+import { useNavigate } from 'react-router-dom';
+import { useAppSelector, useTranslationDefault } from '@/hooks';
```

Comment:
please consolidate and order these inputs from `react-*` to `@ant-design*` to `@` imports to local imports